### DISCUSSION TITLE
Cover from MP3 file

### DISCUSSION THREAD

**Discussion by luk6173** [kagemomiji/airsonic-advanced#454] [Q&A]
Hi all,

I use navidrome, but there is no option to have music separatelly per user. Airsonic Advanced is new for me and I try to figure out, how to get the Cover from each my song(s) shown. Currently I copy my music directly to /var/music folder and do a rescan to see my music in AA but unfortunatelly without covers.

I navidrome I see all covers from my mp3 tracks.

What I have to do to see the song covers?

**Comment by bartmans** [comment]
Check Settings - General.

Cover art files. Add the name you use for covert art (if not already there). Use the same filename for all your art.
Cover art source. Select the correct preference.

Can you not mount your music folder to use in AA or Navidrome? No need to copy over files.

**Comment by luk6173** [comment]
Means I need to add alle filenames from my music tracks?

**Reply by bartmans** [reply]
Do you have all the songs in one folder?

Navidrome uses tags and the embedded art from a mp3 file. 

AA can use 1 folder.jpg file per folder. It relies more on having separate folders per artist - album.

**Comment by luk6173** [comment]
> Do you have all the songs in one folder?

so 1 folder per song, therefore more effort... kind of unattractive

**Comment by bartmans** [comment]
Then stick with navidrome right tools for you it seems. I have my music organised per artist per album. But coverart could use some Rethinking in AA, not ideal yet.
